walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,141 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeUnconsidered() {
+    // No times work with optional person C.
+    //
+    // Events  :       |--A--|     |--B--|
+    //         : |--------------C--------------|     
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+ 
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Optional Event 3", TimeRange.WHOLE_DAY, Arrays.asList(PERSON_C)));
+ 
+    MeetingRequest request =
+        new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B), DURATION_30_MINUTES);
+    request.addOptionalAttendee(PERSON_C);
+ 
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
+        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0800AM, false),
+            TimeRange.fromStartEnd(TIME_0830AM, TIME_0900AM, false),
+            TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, true));
+ 
+    Assert.assertEquals(expected, actual);
```

Comment:
Nit: can just move the expressions in assertEquals instead of giving them variables names. Rationale is when someone reads here they need to go back to find what was expected and actual again. It is more straightforward just spelling out what you are asserting.